_docs/operations/maintenance.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: guide
+title: StorageOS Docs - Maintenance
+anchor: operations
+module: operations/maintenance
+---
+
+# Maintenance
+
+## Node drain
+
+It is recommended to drain a node before any maintenance operation.
```

Comment:
Should have a brief description of what node drain does?  e.g. "Node drain will move any active volumes from the node and re-schedule them on other nodes in a cluster.  This operation will take longer for large volumes."

Reply:
Added clarification: https://github.com/storageos/storageos.github.io/pull/145/commits/784fbe8de26d2f9c6585e52264e750a2a38be230